docs/UsersGuide/source/Quickstart.rst

```diff
@@ -202,6 +216,8 @@ To run Rocoto using the script:
 
 Or to manually call Rocoto: 
 
+NOTE DOM 20220227 on jet at least, rocotorun is not in the path, not even after running "source env/build_jet_intel.env" and "source ../../env/wflow_jet.env"; someone needs to check that on all platforms and insert a code block here to load the rocoto module.
+
 .. code-block:: console
```

Comment:
The build and wflow environment files are not supposed to load the Rocoto module.  That is done on a platform-by-platform basis by the launch_FV3LAM_wflow.sh script:

if [ "$MACHINE" = "CHEYENNE" ]; then
  module use -a /glade/p/ral/jntp/UFS_SRW_app/modules/
  module load rocoto
elif [ "$MACHINE" = "ORION" ]; then
  module purge
  module load contrib rocoto
elif [ "$MACHINE" = "WCOSS_DELL_P3" ]; then
  module purge
  module load lsf/10.1
  module use /gpfs/dell3/usrx/local/dev/emc_rocoto/modulefiles/
  module load ruby/2.5.1 rocoto/1.2.4
elif [ "$MACHINE" = "WCOSS_CRAY" ]; then
  module purge
  module load xt-lsfhpc/9.1.3
  module use -a /usrx/local/emc_rocoto/modulefiles
  module load rocoto/1.2.4
elif [ "$MACHINE" = "GAEA" ]; then
  module use /lustre/f2/pdata/esrl/gsd/contrib/modulefiles
  module load rocoto/1.3.3
else
  module purge
  module load rocoto
fi

Reply:
So, based on this we have two options. I can add in these module loads for each platform or we can remove the */3 * * * * cd /glade/p/ral/jntp/$USER/expt_dirs/test_CONUS_25km_GFSv15p2 && /glade/p/ral/jntp/tools/rocoto/rocoto-1.3.1/bin/rocotorun -w FV3LAM_wflow.xml -d FV3LAM_wflow.db -v 10  (which happens to be what I use all the time!) and just leave */3 * * * * cd /glade/p/ral/jntp/$USER/expt_dirs/test_CONUS_25km_GFSv15p2 && ./launch_FV3LAM_wflow.sh

Reply:
BTW, have you considered explicitly listing Hera and Jet in that list rather than the just leaving it in the else block? Not a big deal for this release, but may be something to consider for the future.

Reply:
@jwolff-ncar, you're correct.  Hera and Jet should be explicitly listed.  We can add that to our list of post-release changes.

Can we just add a caveat to the rocotorun/rocotoboot commands listed and say (the Rocoto module must be in your path)?  Again, these commands (and the crontab example) are only necessary if the user does not use the launch script or the crontab option in config.sh.

Reply:
I added in the module loads for the different platforms for now. If there is disagreement we can discuss tomorrow.